### PR TITLE
added styles in line 592

### PR DESCRIPTION
**Pull request by osbentech**
here is a link to my portfolio-project https://osbentech.github.io/Portfolio/

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -591,7 +591,7 @@ body {
```
Comment (vechicin) [inline styles.css:599]: When I go into the mobile version, I'm experiencing a horizontal scroll (I'm using Google Chrome, for reference). Please make sure you fix it as horizontal scrolls make for bad UI/UX/

Screen Shot 2022-07-18 at 12 59 20

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @osbentech ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good use of GitHub Pages!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vechicin** [comment]
https://github.com/osbentech/Portfolio/blob/41967c5ae7c14c603fd42eae8bff19a3c703d39e/README.md?plain=1#L1

Please make sure you add a Live Demo link to your README as it is a project requirement.

Screen Shot 2022-07-18 at 13 01 35

**Review by sumairq — APPROVED** [review]
### Hi @osbentech  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ The portfolio is deployed to github pages
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -591,7 +599,7 @@ body {
 
   .middle-section-container {
     display: flex;
-    width: 85%;
+    width: 86%;
     height: 40%;
     background-color: white;
     border-bottom-left-radius: 20px;
```
Comment (iAmWaris97) [inline styles.css:605]: Nice job!
I would suggest you change the order of the images and details of the cards as of Figma design. What do you think?

**Review by iAmWaris97 — COMMENTED** [review]
Hi @osbentech!

Nice job so far!

I liked the styles, however, I would suggest you some changes as commented below.

Cheers and happy coding.

**Pull request merged**